pandas/tests/test_errors.py

```diff
@@ -110,3 +110,8 @@ def test_AbstractMethodError_classmethod():
     xpr = "This method must be defined in the concrete class Foo"
     with pytest.raises(AbstractMethodError, match=xpr):
         Foo().method()
+
+    xpr = "methodtype must be one of {'method', 'classmethod', "
+          "'staticmethod', 'property'}, got unknownmethod instead."
```

Comment:
final nitpick, the rest looks good (pending green)
```
got 'unknownmethod' instead
```
(same for where the error is raised)

Reply:
https://github.com/pandas-dev/pandas/pull/51484/commits/58a54bae6292cff2b61ea3996db59c24fc62f779